cli/src/pcluster/config/cluster_config.py

```diff
@@ -1183,6 +1203,79 @@ def __init__(
         self.on_node_updated = Resource.init_param(on_node_updated)
 
 
+class LoginNodeImage(Resource):
+    """Represent the configuration of LoginNode Image."""
+
+    def __init__(self, custom_ami: str, **kwargs):
+        super().__init__()
+        self.custom_ami = Resource.init_param(custom_ami)
+
+    def _register_validators(self, context: ValidatorContext = None):  # noqa: D102 #pylint: disable=unused-argument
+        if self.custom_ami:
+            self._register_validator(CustomAmiTagValidator, custom_ami=self.custom_ami)
+
+
+class LoginNodeSsh(Ssh):
```

Comment:
Let's handle this like the IAM scenario (i.e. Inheritance) since the `Ssh` section for `LoginNodes` doesn't have an Allowed IPs parameter for now.

The Allowed IPs parameter might be an implied value if we keep it as it is now but the schema should try to match the resource to ensure it's not unintentionally used downstream.

Reply:
Done!